blackbox/docs/sql/information_schema.txt

```diff
@@ -27,63 +27,41 @@ settings like the ``number of shards`` or ``number of replicas``.
     ... ) partitioned by (date);
     CREATE OK, 1 row affected (... sec)
 
-::
-
-    cr> select table_schema, table_name, number_of_shards, number_of_replicas
```

Comment:
Yeah, maybe it would be better to retain this information and have the listing of the columns underneath it.